protos/action/action.proto

```diff
@@ -177,6 +183,18 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+message DoOrbitRequest {
+    float radius_m = 1; // Radius of circle (in meters)
+    float velocity_ms = 2; // Tangential velocity (in m/s)
+    float yaw_behavior = 3; // Yaw behavior of vehicle (ORBIT_YAW_BEHAVIOUR)
+    double latitude_x = 5; //Center point latitude (no mav_frame) / X coord to mav_frame. NAN: current
+    double longitude_y = 6; //Center point longitude (no mav_fram) / Y coord to mav_frame. NAN: current
+    double altitude_z = 7; //Center point altitude (MSL, no mav_frame) / Z coord to mav_frame. NAN: current
```

Comment:
```suggestion
    double latitude_x = 5; // Center point latitude (no mav_frame) / X coord to mav_frame. NAN: current
    double longitude_y = 6; // Center point longitude (no mav_fram) / Y coord to mav_frame. NAN: current
    double altitude_z = 7; // Center point altitude (MSL, no mav_frame) / Z coord to mav_frame. NAN: current
```

I'm not sure I get that. So it's either a latitude in degrees, or an x coordinate in meters w.r.t. to some local frame?

Reply:
I was just trying to abbreviate the description as in this example:

"Center point latitude (if no MAV_FRAME specified) / X coordinate according to MAV_FRAME. NaN: Use current vehicle position or current center if already orbiting."

Reply:
Where does that example come from?